features/environment.py

```diff
@@ -141,33 +174,46 @@ def _capture_container_as_image(container_name: str, image_name: str) -> None:
     subprocess.run(["lxc", "publish", container_name, "--alias", image_name])
 
 
-def create_trusty_uat_lxd_image(context: Context) -> None:
-    """Create a trusty lxd image with ubuntu-advantage-tools installed
+def create_uat_lxd_image(context: Context, series: str) -> None:
+    """Create a given series lxd image with ubuntu-advantage-tools installed
 
     This will launch a container, install ubuntu-advantage-tools, and publish
-    the image.  The image's name is stored in context.image_name for use within
-    step code.
+    the image.    The image's name is stored in context.series_image_name for
+    use within step code.
 
     :param context:
-        A `behave.runner.Context`; this will have `image_name` set on it.
+        A `behave.runner.Context`;  this will have `series.image_name` set on
+        it.
+    :param series:
+       A string representing the series name to create
     """
 
-    def image_cleanup() -> None:
-        if context.config.image_clean:
-            subprocess.run(["lxc", "image", "delete", context.image_name])
-        else:
-            print("Image cleanup disabled, not deleting:", context.image_name)
-
     if context.reuse_container:
-        print(" Reusing the existent container: ", context.reuse_container)
-    else:
-        now = datetime.datetime.now()
-        context.image_name = "behave-image-" + now.strftime("%s%f")
-        build_container_name = "behave-image-build-" + now.strftime("%s%f")
-        launch_lxd_container(context, "ubuntu:trusty", build_container_name)
-        _install_uat_in_container(build_container_name)
-        _capture_container_as_image(build_container_name, context.image_name)
-        context.add_cleanup(image_cleanup)
+        container_series = lxc_get_series(context.reuse_container)
+        if container_series == series:
+            print(
+                "\n Reusing the existent container: ", context.reuse_container
+            )
+            print(
+                """\n You are providing a %s container.
+                Make sure you are running this series tests."""
```

Comment:
Can this message print "Make sure you provided -tags=series.%s" % series

Reply:
Ok. I've added `For instance: --tags=series.{series}` because it is not an obligation to run behave providing `--tags=...`. If you don't add this, it will run all tests.